game/selenium_tests/base.py

```
@@ -0,0 +1,93 @@
+    print(f"  {CYAN}ℹ  {msg}{RESET}")
+
+
+def log_warn(msg):
+    print(f"  {YELLOW}⚠  {msg}{RESET}")
+
+
+class BaseE2ETest(LiveServerTestCase):
+    """Shared Selenium setup, teardown, and helper methods."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.12)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)